extensions/vscode/src/extension.ts

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+import * as net from 'net';
+
+import {Trace} from 'vscode-jsonrpc';
+import { window, workspace, commands, ExtensionContext, Uri } from 'vscode';
+import { LanguageClient, LanguageClientOptions, StreamInfo, Position as LSPosition, Location as LSLocation } from 'vscode-languageclient';
+
+export function activate(context: ExtensionContext) {
+    // The server is a started as a separate app and listens on port 5007
+    let connectionInfo = {
+        port: 5007
+    };
+    let serverOptions = () => {
+        // Connect to language server via socket
+        let socket = net.connect(connectionInfo);
```

Comment:
please note: for production you should switch back to a language server started via subprocess (bash script, uber jar) here. the socket mode is imho only useful for local debugging